packages/core/crates/core-strategy/src/auto_redeeming.rs

```diff
@@ -55,7 +54,8 @@ impl<Db: HoprCoreEthereumDbActions + 'static> SingularStrategy for AutoRedeeming
     async fn on_acknowledged_ticket(&self, ack: &AcknowledgedTicket) -> crate::errors::Result<()> {
         if !self.cfg.redeem_only_aggregated || ack.ticket.is_aggregated() {
             info!("{self} strategy: auto-redeeming {ack}");
-            let _ = redeem_ticket(self.db.clone(), ack.clone(), self.tx_sender.clone()).await?;
+            let rx = redeem_ticket(self.db.clone(), ack.clone(), self.tx_sender.clone()).await?;
+            std::mem::drop(rx); // The Receiver is not intentionally awaited here
```

Comment:
Make sure no `Sender::start_send` is used anywhere on the other end without poll ready.

Reply:
This is a oneshot channel, so this is not an issue.